packages/wonder-blocks-button/src/components/button-unstyled.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
+import {Link, useInRouterContext} from "react-router-dom-v5-compat";
+
+import {addStyle} from "@khanacademy/wonder-blocks-core";
+import {isClientSideUrl} from "@khanacademy/wonder-blocks-clickable";
+import {StyleSheet} from "aphrodite";
+import {ButtonProps} from "../util/button.types";
+
+const StyledA = addStyle("a");
+const StyledButton = addStyle("button");
+const StyledLink = addStyle(Link);
+
+type Props = Omit<ButtonProps, "children"> & {
+    /**
+     * The button content.
+     */
+    children: React.ReactNode;
+};
+
+const ButtonUnstyled: React.ForwardRefExoticComponent<
```

Comment:
Do you think it would be helpful to have visual regression tests for ButtonUnstyled on it's own?

Reply:
I think it should be fine to leave it as it is as the styles are fairly simple.